mail/libvmime/Portfile

```diff
@@ -50,6 +50,11 @@ configure.args      -DVMIME_BUILD_DOCUMENTATION=OFF \
                     -DVMIME_CHARSETCONV_LIB=icu \
                     -DVMIME_TLS_SUPPORT_LIB=gnutls
 
+# Unfortunately, it uses an archaic TargetArch module,
+# which requires a configure arg to unbreak ppc builds.
+configure.args-append \
+                    -Dppc_support=ON
+
```

Comment:
Arguably, this can be turned on just on the PPC builds.

Reply:
It has zero effect on anything else, it simply prevents CMake from pretending the arch does not exist.